### ISSUE TITLE
wpf .net core 3.0 Failed to resolve System.Reflection.BindingFlags

### ISSUE THREAD

**Issue by AR1ES** [obfuscar/obfuscar#249]
I tried to configure obfuscar on a desktop .net core 3.0 DLL , but I get an error:

```Loading assemblies...Extra framework folders: Done.
Hiding strings...
Renaming:  fields...Parameters...Properties...Events...Methods...Types...Done.
Saving assemblies...
Unhandled Exception: Mono.Cecil.ResolutionException: Failed to resolve System.Reflection.BindingFlags
   at Mono.Cecil.Mixin.CheckedResolve(TypeReference self)
   at Mono.Cecil.MetadataBuilder.GetConstantType(TypeReference constant_type, Object constant)
   at Mono.Cecil.MetadataBuilder.AddConstant(IConstantProvider owner, TypeReference type)
   at Mono.Cecil.MetadataBuilder.AddParameter(UInt16 sequence, ParameterDefinition parameter, ParamTable table)
   at Mono.Cecil.MetadataBuilder.AddParameters(MethodDefinition method)
   at Mono.Cecil.MetadataBuilder.AddMethod(MethodDefinition method)
   at Mono.Cecil.MetadataBuilder.AddMethods(TypeDefinition type)
   at Mono.Cecil.MetadataBuilder.AddType(TypeDefinition type)
   at Mono.Cecil.MetadataBuilder.AddTypes()
   at Mono.Cecil.MetadataBuilder.BuildTypes()
   at Mono.Cecil.MetadataBuilder.BuildModule()
   at Mono.Cecil.MetadataBuilder.BuildMetadata()
   at Mono.Cecil.ModuleWriter.<>c.<BuildMetadata>b__2_0(MetadataBuilder builder, MetadataReader _)
   at Mono.Cecil.ModuleDefinition.Read[TItem,TRet](TItem item, Func`3 read)
   at Mono.Cecil.ModuleWriter.Write(ModuleDefinition module, Disposable`1 stream, WriterParameters parameters)
   at Mono.Cecil.ModuleWriter.WriteModule(ModuleDefinition module, Disposable`1 stream, WriterParameters parameters)
   at Obfuscar.Obfuscator.SaveAssemblies(Boolean throwException)
   at Obfuscar.Obfuscator.RunRules()
   at Obfuscar.Program.Main(String[] args)
```

**Comment by monty241** [comment]
Maybe same problem as #248  ?

**Comment by AR1ES** [comment]
No.
Version of obfuscar is already 2.2.25.0

**Comment by WayneWinter** [comment]
Also have this issue in 2.2.31, then upgraded to 2.2.32, still same problem, thanks

**Comment by M1raclee** [comment]
I also encountered this exception. It was caused by a constant of type BindingFlags.

**Solution:**
Add to obfuscar.xml <AssemblySearchPath path="path_to_folder_with_System.Reflection.dll">

**Comment by lextm** [comment]
Will be resolved by #553
